.github/workflows/update-template-snapshots.yml

```diff
@@ -0,0 +1,160 @@
+name: Update Template snapshots from a comment
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  prepare-matrix:
+    name: Retrieve all template features
+    # Only run if comment is on a PR with the main repo, and if it contains the magic keywords
+    if: >
+      contains(github.event.comment.html_url, '/pull/') &&
```

Comment:
do we need this?

Reply:
I think so? Otherwise, it would try to run on comments to issues too